pkg/kubectl/rollout_status.go

```diff
@@ -34,6 +34,8 @@ func StatusViewerFor(kind unversioned.GroupKind, c internalclientset.Interface)
 	switch kind {
```

Comment:
Separate commit with kubectl changes - you could squash all kubectl changes into one commit if you want.